packages/component-base/src/tooltip-controller.d.ts

```diff
@@ -23,6 +23,12 @@ type TooltipPosition =
  * A controller that manages the slotted tooltip element.
  */
 export class TooltipController extends SlotController {
+  /**
+   * An HTML element to attach the tooltip to.
```

Comment:
This should clarify how the property relates to aria attributes specifically.

Reply:
Updated to mention the `aria-describedby` attribute.